iam-password-policy/change-password-policy.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python3
+
+from copy import Error
+import boto3
+from botocore.exceptions import ClientError
+import os
+import logging
+from botocore.session import Session
+
+
+
+
+
+def main(args, logger):
+    '''Executes the Primary Logic of the Fast Fix'''
+
+    # If they specify a profile use it. Otherwise do the normal thing
+    if args.profile:
+        session = boto3.Session(profile_name=args.profile)
+    else:
+        session = boto3.Session()
+
+    try:
+        response = setPasswordPolicy(args, client= session.client(service_name='iam'))
+        logging.debug("Response:"+str(response))
+        logging.info("Response: Successfuly updated password policy")
+    except Error as err:
```

Comment:
You want to catch either Exception or ClientError - catching Error didn't work for my expired creds

Reply:
Fixed, Listened to autocomplete, normally do exception as err